Student.js

```diff
@@ -188,6 +188,12 @@ function std() {
 }
 
 function res() {
+for(k=0;k<100;k++){
+if(i==200)

```

Comment:
Declare the variable 'i' before using it in the loop condition to avoid reference errors. [important]